storybooknative/src/Text/__tests__/index.test.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import * as React from "react";
+import renderer from "react-test-renderer";
+
+import Text from "../index";
+import { TYPE_OPTIONS, SIZE_OPTIONS, WEIGHT_OPTIONS, ALIGN_OPTIONS } from "../consts";
+
+describe("Text", () => {
+  const text = "Lorem ipsum";
+
+  Object.keys(TYPE_OPTIONS).forEach(type => {
+    Object.keys(SIZE_OPTIONS).forEach(size => {
+      Object.keys(WEIGHT_OPTIONS).forEach(weight => {
+        Object.keys(ALIGN_OPTIONS).forEach(alignment => {
+          test(`${type}-${size}-${weight}-${alignment} should match the snapshot`, () => {
+            expect(
+              renderer
+                .create(
+                  <Text type={type} size={size} weight={weight} align={alignment}>
```

Comment:
@denbalogh, you're passing the `key` to the component and not the `value`. I'm curious why the Flow doesn't throw an error? cc @RobinCsl

Reply:
Actually, I wrote this and you're right, it should not be the key but the corresponding value. That's really odd.